src/test/java/com/segment/analytics/android/integrations/comscore/ComScoreTest.java

```diff
@@ -504,36 +474,35 @@ public void videoContentStartedWithDigitalAirdate() {
     expected.put("c4", "*null");
     expected.put("c6", "*null");
 
-    Mockito.verify(streamingAnalytics).notifyPlay(70);
-    Mockito.verify(playbackSession).setAsset(expected);
+
+    Mockito.verify(streamingAnalytics).startFromPosition(70);
+    Mockito.verify(streamingAnalytics).notifyPlay();
+    Mockito.verify(streamingAnalytics,atLeast(2))
```

Comment:
Should we be setting the same metadata twice? If I understand correctly, this is what this is testing for, which seems unexpected.

Reply:
This test passes by checking that the method is called at least twice, but I think we should check only once for clarity. The first invocation of `setMetadata` occurs upon invocation of `setupWithVideoPlaybackStarted();`.

So,  `Mockito.verify(streamingAnalytics,atLeast(1)).setMetadata(refEq(getContentMetadata(expected)));`

Reply:
Ohh Yeah that's right